shell/imports/server/migrations.js

```diff
@@ -550,6 +554,9 @@ const consolidateOrgSettings = function (db, _backend) {
     ldap: {
       enabled: orgLdap ? orgLdap.value : false,
     },
+    ldap: {
```

Comment:
Is this supposed to be for OIDC here? It looks like it's just the above three lines duplicated.